docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -1,229 +1,76 @@
-    dependencies as the job had when it was launched.
-
-
 Update: ``klio-job.yaml``
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Under ``pipeline_options``, add the key ``setup_file`` and set the value to ``setup.py``. This
-tells Klio and Dataflow to not use the FnAPI; rather, to use ``setup.py`` as the mechanism for
-packaging the job.
+Under ``pipeline_options``:
+
+1. **DELETE** the key ``setup_file``.
+2. **ADD** the list key ``experiments`` under ``pipeline_options``, containing the item ``beam_fn_api``. Using the ``beam_fn_api`` experiment in conjunction with setting the ``worker_harness_container_image`` tells Klio and Dataflow to use FnAPI rather than the setup file to package the job.
 
 .. collapsible:: Minimal Example ``klio-job.yaml``
 
     .. code-block:: yaml
 
         job_name: my-job
         pipeline_options:
-          setup_file: setup.py # relative to repo root
+          # NOTE! setup_file is absent
+          experiments:
+            - beam_fn_api
           worker_harness_container_image: gcr.io/my-project/my-job-image
           runner: DataflowRunner
           # <-- snip -->
 
-.. attention::
-
-    **The worker image is still needed!**
-
-    Do not remove the ``worker_harness_container_image`` value under ``pipeline_options``. Klio
-    uses the image as a `"driver"`_ for Beam.
-
 
 Update: ``Dockerfile``
 ^^^^^^^^^^^^^^^^^^^^^^
 
 Required Changes
 ~~~~~~~~~~~~~~~~
 
-1. **ADD** ``klio-job.yaml`` to be copied into ``/usr/src/app``.
-
-    .. collapsible:: Why is this needed?
 
-        We need to include Klio's configuration, but when creating a package of the job, the configuration must be within the same directory ``setup.py`` is in (subdirectories are fine). Relatedly, multi-configuration is not yet supported without the FnAPI since Klio expects the job configuration in a location that we can't manipulate with the ``setup.py`` approach.
-
-2. **ADD** the newly required files to be copied over - ``setup.py`` and ``MANIFEST.in`` - into the working directory, ``/usr/src/app``.
-
-    .. collapsible:: Why is this needed?
-
-        ``setup.py`` and ``MANIFEST.in`` are needed to tell Klio and Dataflow how to build your pipeline as a Python package (i.e. what Python and non-Python files to include) since you're no longer using a Docker image as a "package" for your job.
-
-3. **DOUBLE CHECK** any non-Python files needed for the job, e.g. models, JSON schemas, etc, are copied into the working directory, ``/usr/src/app``.
-
-    .. collapsible:: Why is this needed?
-
-        Klio packages up your job to be installed (for unit tests, audits, and running on the direct runner), and to be uploaded to Dataflow locally on the job's worker image. Therefore, the Docker image needs to have all the required Python and non-Python files to run the job.
-
-4. **ADD** the following line to the end of the file: ``RUN pip install .``
+1. **UPDATE**  ``RUN pip install .`` to ``RUN pip install -r job-requirements.txt``
```

Comment:
See comment below re: where to put `RUN pip install -r job-requirements.txt`. It might be easier to have two separate steps instead of one **UPDATE** step.